src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -301,12 +301,14 @@ def load_arguments(self, _):
         c.argument('target_database_account_name', options_list=['--target-database-account-name', '-n'], help='Name of the new target Cosmos DB database account after the restore')
         c.argument('account_name', completer=None, options_list=['--account-name', '-a'], help='Name of the source Cosmos DB database account for the restore', id_part=None)
         c.argument('restore_timestamp', options_list=['--restore-timestamp', '-t'], action=UtcDatetimeAction, help="The timestamp to which the account has to be restored to.")
-        c.argument('location', arg_type=get_location_type(self.cli_ctx), help="The location of the source account from which restore is triggered. This will also be the write region of the restored account")
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), help="In case of same region restore, this is the location of the source account from which restore is triggered. This will also be the write region of the restored account. But for cross region restore this would be the target location of the restored account.")
```

Comment:
"This is the write region of the restored account.  This is also the location of the source account the backups are located if source_backup_location is not provided."

Reply:
"This is the write region of the restored account. This is also the location of the source account where its backups are located if source_backup_location is not provided."